### ISSUE TITLE
Develop process and/or tool for upgrading to Polymer 2 and PolymerDecorators

### ISSUE THREAD

**Issue by kito99** [nippur72/PolymerTS#110]
Polymer has added support for TypeScript with their PolymerDecorators project (Polymer 2 only). This issue is for providing a smooth migration path for existing Polymer 1.0/PolymerTS users.

**Comment by pme123** [comment]
> This issue is for providing a smooth migration path for existing Polymer 1.0/PolymerTS users.

Is here any progress?

**Comment by kito99** [comment]
Yes, there is some progress; @keithstric  is working on a converter: https://github.com/RedPillNow/polymerts-to-polymer-decorators.
